### ISSUE TITLE
Give user option to choose multiple meds

### ISSUE THREAD

**Issue by tmthyjames** [tmthyjames/Achoo#20]
On sign up page, allow user to choose more than just the inhaler. Add a plus sign sign button to add more options.

**Comment by sparklespdx** [comment]
I definitely want to add a couple of options for my own use.

**Comment by tmthyjames** [comment]
I agree. Plus there are different types of inhalers and later on when there's a lot more data from multiple users, it may help the predictive power of the models to differentiate between the different types of inhalers. This is way down the line though, I think.

Also, eventually I'd like to experiment with doing this same thing but with allergies. Since allergies and asthma are fundamentally very different, this may take some brainstorming.
